doc/ale-cpp.txt

```diff
@@ -190,4 +190,23 @@ g:ale_cpp_gcc_options                                   *g:ale_cpp_gcc_options*
 
 
 ===============================================================================
+clang-format                                              *ale-cpp-clangformat*
+
+g:ale_c_clangformat_executable                 *g:ale_c_clangformat_executable*
```

Comment:
I think here you can just write `See |ale-c-clangformat|` and a short message about the options being the same.